resources/frontend-development.md

```diff
@@ -8,3 +8,11 @@ Here is a list of useful websites for learning and mastering CSS:
 - [freeCodeCamp](https://www.freecodecamp.org/learn/)
 - [Flexbox Frogy](https://flexboxfroggy.com/)
 - [CSS Grid Garden](https://cssgridgarden.com/)
+
+## JavaScript Library
+
+A javaScript library is a toolbox of pre-written code designed for a specific task. In other words,it's a collection of reusable functions and features framed or crafted to make certain programming tasks easier. For a beginner, think of it as a set of tools that you can use to build a web application without having to create everything from scratch. An example of a JavaScript library is React.
+
+Here is a useful YouTube video tutorial to learn and master React:
+
+- [freeCodeCamp](https://www.youtube.com/watch?v=bMknfKXIFA8&t=853s)
```

Comment:
@RitaDee I would like to see only the link to be a bit more informative with it's title, as we have freecodecamp linked elsewhere. 

I like the JavaScript library description and I liked the additional description for the course written by @adiati98 . As we have a new issue to handle descriptions I leave it up to @RitaDee to decide. Maybe you'd like to do more PRs ;)

Hope that makes sense! Thanks all, and thank you @RitaDee for your contribution and patience with our input. :)

Reply:
> @RitaDee I would like to see only the link to be a bit more informative with it's title, as we have freecodecamp linked elsewhere.
> 
> I like the JavaScript library description and I liked the additional description for the course written by @adiati98 . As we have a new issue to handle descriptions I leave it up to @RitaDee to decide. Maybe you'd like to do more PRs ;)
> 
> Hope that makes sense! Thanks all, and thank you @RitaDee for your contribution and patience with our input. :)

It does make sense, @mariahlaqua. I will create a PR for this in the new issue you created. Thank you.